style.css

```diff
@@ -151,10 +189,37 @@ section .icon-right {
   cursor: pointer;
 }
 
+.pic-container .cards-pic {
+  width: 100%;
+  height: 85%;
+}
+
```

Comment:
[OPTIONAL] I would recommend you fix your popup image width. This is the way it's currently looking on my end.

<img width="1308" alt="Screen Shot 2022-06-15 at 8 15 46" src="https://user-images.githubusercontent.com/82426682/173836062-28c2da92-b909-415c-9f4c-80222f68173d.png">

Reply:
yes, the image was difficult to style. it stretches. If you know any procedure to help me style it, I will be grateful